src/arr/compiler/cli-module-loader.arr

```diff
@@ -425,16 +425,18 @@ fun build-runnable-standalone(path, require-config-path, outfile, options) block
       end
 
       when options.collect-times: stats.set-now("standalone", time-now()) end
-      ans = MS.make-standalone(program.natives, program.js-ast,
-        JSON.j-obj(config.freeze()).serialize(), options.standalone-file)
-      when options.collect-times block:
-        standalone-end = time-now() - stats.get-value-now("standalone")
-        stats.set-now("standalone", [list: "Outputing JS: " + tostring(standalone-end) + "ms"])
-        for SD.each-key-now(key from stats):
-          print(key + ": \n" + stats.get-value-now(key).join-str(", \n") + "\n")
+
+      callback = lam(_):
+        when options.collect-times block:
+          standalone-end = time-now() - stats.get-value-now("standalone")
+          stats.set-now("standalone", [list: "Outputing JS: " + tostring(standalone-end) + "ms"])
+          for SD.each-key-now(key from stats):
+            print(key + ": \n" + stats.get-value-now(key).join-str(", \n") + "\n")
+          end
         end
       end
-      ans
+      MS.make-standalone(program.natives, program.js-ast,
+        JSON.j-obj(config.freeze()).serialize(), options.standalone-file, callback)
```

Comment:
Oh, nice. Then when that gets merged in we can get rid of some ugly callback code I had to add